src/components/action-sheet/action-sheet.scss

```diff
@@ -61,4 +61,8 @@ action-sheet {
       transform: translateY(calc(-100% + 8px));
     }
   }
+  .safe-top-area {
+    padding-top: calc(10px + env(safe-area-inset-top));
```

Comment:
10px is applied to desktop too. Is that the idea?

Reply:
it is on purpose. It looks more similar to the design